script/Ciel/Day 13/13_01_01_CIEL13_3B.txt

```diff
@@ -920,7 +920,7 @@ Nothing more than a convenient marionette,
 -- Page 186, Offset 33717.
 -- 吸血鬼に消費されて捨てられた、どこの誰とも知らない子供。それが貴方の正体です」
 That is who you really are:
- a child from who-knows-where, devoured and cast away by a vampire."
+ a child from who knows where, devoured and cast away by a vampire."
```

Comment:
Pretty sure it SHOULD have dashes, but finding documentation is hard!

https://muppet.fandom.com/wiki/Hole_to_Who-Knows-Where

MUPPET FANDOM WIKI GO